public/post-robot-example.html

```diff
@@ -0,0 +1,111 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Map embed test</title>
+  <script src="https://cdn.jsdelivr.net/npm/post-robot@8.0.28/dist/post-robot.min.js"></script>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@900&display=swap" rel="stylesheet"> 
+</head>
+<style>
+  body {
+    margin: 0;
+    font-family: 'Lato', sans-serif;
+  }
+  iframe,
+  section {
+    width: 49vw;
+    height: 100vh;
+  }
+  section {
+    float: right;
+  }
+  div {
+    display: flex;
+    flex-direction: column;
+    max-width: 300px;
+    margin-bottom: 50px;
+  }
+  input {
+    margin-bottom: 30px;
+  }
+  label {
+    position: relative;
+  }
+  label > span{
+    position: absolute;
+    top: -20px;
+  }
+  button {
+    cursor: pointer;
+  }
+</style>
+<body>
+  <iframe id="map-iframe" src="http://localhost:3000/map?listeners=true" frameborder="0"></iframe>
+  <section>
+    <p>To handle interactions with the map iframe we are using <a href="https://github.com/krakenjs/post-robot">postRobot</a> library</p>
+    <p>This functions are triggering the postRobot.send() method.</p>
+    <p>Check the code of <a href="https://github.com/Vizzuality/half-earth-v3/blob/feature/map-iframe-constatnt-contact-layers/public/post-robot-example.html">this</a> page in this file</p>
+    <div>
+      <p>Check available layers names in this <a href="https://github.com/Vizzuality/half-earth-v3/blob/master/src/constants/layers-slugs.js">file</a></p>
```

Comment:
I was stupid enough to paste the name of the const (`SA_AMPHIB_RARITY`) instead of value `amphib-rarity-sa`, so maybe we can write a small hint about that here. Maybe we can change the `Check available layers names...` to `Check available 
 layers slugs...`